src/lib/util/extract-user.test.ts

```diff
@@ -22,11 +23,11 @@ describe('extractUsernameFromUser', () => {
     test('Should return "unknown" if neither email nor username exists', () => {
```

Comment:
I guess the title of the test is slightly incorrect after this change :)

Reply:
Spot on!